### ISSUE TITLE
Simplex noise

### ISSUE THREAD

**Issue by MSGhero** [HaxeFlixel/flixel-addons#266]
I think we agreed on putting a Simplex noise generator in addons, but where in the package structure should it go?

I still need to clean it up a bit before submitting.

**Comment by Gama11** [comment]
Well, where do you think it should go?

Does it count as an effect? https://github.com/HaxeFlixel/flixel-addons/tree/dev/flixel/addons/effects

**Comment by MSGhero** [comment]
Not really, it's the math behind a potential effect, but not the effect itself. `addons/math` might be more appropriate, but that seems fairly niche.

**Comment by sruloart** [comment]
Maybe we can use an "algorithms" folder? there are plenty of niche gamedev-related algorithms to fill it up with :) it's also somewhat more meaningful than just plain old "math"

**Comment by MSGhero** [comment]
Oh boy I'd have a lot of things to add to that
